server/Justfile

```diff
@@ -13,3 +13,8 @@ fmt:
     goimports -w .
     go fmt ./...
 
+psql:
+    @PGPASSWORD=$POSTGRES_PASSWORD psql -h localhost -U $POSTGRES_USER $POSTGRES_DB
```

Comment:
@fedeizzo this only starts an SQL shell in the local database, right?

Reply:
Yes, it runs locally on your machine. We can run the same command inside the pg container to avoid `psql` as a dep in the nix flake, wdyt?

Reply:
I'd say we can leave this to each developer, I for instance usually connect through a UI like DBevear and not through the command line. Maybe mentioning this command in the Contribution Guide is sufficient?